src/test/java/net/snowflake/ingest/streaming/internal/it/ColumnNamesIT.java

```diff
@@ -13,17 +17,45 @@
 import net.snowflake.ingest.streaming.OpenChannelRequest;
 import net.snowflake.ingest.streaming.SnowflakeStreamingIngestChannel;
 import net.snowflake.ingest.streaming.internal.datatypes.AbstractDataTypeTest;
+import net.snowflake.ingest.utils.Constants;
 import net.snowflake.ingest.utils.SFException;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runners.Parameterized;
 
 public class ColumnNamesIT extends AbstractDataTypeTest {
   private static final int INGEST_VALUE = 1;
 
+  @Parameterized.Parameters(
+      name = "enableIcebergStreaming={0}, compressionAlgorithm={1}, icebergSerializationPolicy={2}")
+  public static Object[][] parameters() {
+    return new Object[][] {
+      // TODO: Enable this after Iceberg testing is set up on sfctest0 GCP / Azure
+      //      {true, "ZSTD", Constants.IcebergSerializationPolicy.COMPATIBLE},
+      //      {true, "ZSTD", Constants.IcebergSerializationPolicy.OPTIMIZED},
+      {false, "GZIP", null},
+      {false, "ZSTD", null},
```

Comment:
why do we need gzip and std testing in this IT? This IT is only validating the column names / EP handling, file compression doesn't need to be tested in this one.

Reply:
Will split this into two separate tests in later PR.